### DISCUSSION TITLE
Latest update to 2.8 for HA says "Don't update?"

### DISCUSSION THREAD

**Discussion by starman2k01** [alexbelgium/hassio-addons#1377] [Q&A]
I assume this is the ARM update? What happens if I go ahead and update?
Thanks

**Comment by alexbelgium** [comment]
Ahah, it is just because I won't be able to support for 1 week :) so indeed it should be working fully but I set it that way so that only people who would be mindful of potential issues would update :)

**Comment by starman2k01** [comment]
Great, thanks for the update! Loaded just fine. Accessing gets:
No route or file found for resource GET: /
Which I guess is normal?

**Comment by alexbelgium** [comment]
I think the previous version had the same but it still worked
